tensorrt_llm/_torch/attention_backend/sparse/dsa.py

```
@@ -1068,7 +1060,7 @@ def sparse_attn_indexer(
     def forward(self, qr: torch.Tensor, hidden_states: torch.Tensor,
                 metadata: DSAtrtllmAttentionMetadata,
                 position_ids: torch.Tensor):
-        quant_block_size = metadata.kv_cache_manager.quant_block_size
+        metadata.kv_cache_manager.quant_block_size
```

Ruff (0.14.1)

Found useless expression. Either assign it to a variable or remove it.

(B018)